docs/taskruns.md

```diff
@@ -333,11 +333,15 @@ spec:
           type: git
     steps:
       - name: build-and-push
+<<<<<<< HEAD
         image: gcr.io/kaniko-project/executor
         # specifying DOCKER_CONFIG is required to allow kaniko to detect docker credential
         env:
           - name: "DOCKER_CONFIG"
             value: "/builder/home/.docker/"
+=======
+        image: gcr.io/kaniko-project/executor:v0.9.0
+>>>>>>> 85e787e8... Add a specific verion for the image gcr.io/kaniko-project/executor
```

Comment:
i think something went wrong with the merge conflict resolution here @houshengbo ! I think what you need to do is update line 337 with `v0.9.0`, remove `<<<<<<< HEAD` and remove everything from `=======` to `>>>>>>> 85e787e8... Add a specific verion for the image gcr.io/kaniko-project/executor`

Reply:
oh soory. forgot to resolve it.